### ISSUE TITLE
doの意味が全体的に逆になっています。

### ISSUE THREAD

**Issue by megascus** [dotnet/docs.ja-jp#118]
doが"しないで"と翻訳されています。

---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: 1cd21cac-a7e8-13ca-d940-3447312871ca
* Version Independent ID: 7f3307e9-5563-e28f-cdad-9b6e33358c6b
* Content: [型のメンバーの名前](https://docs.microsoft.com/ja-jp/dotnet/standard/design-guidelines/names-of-type-members)
* Content Source: [docs/standard/design-guidelines/names-of-type-members.md](https://github.com/dotnet/docs.ja-jp/blob/live/docs/standard/design-guidelines/names-of-type-members.md)
* GitHub Login: @rpetrusha
* Microsoft Alias: **ronpet**

**Comment by CeciAc** [comment]
こんにちは、@ megascus,

フィードバックをお送りいただきありがとうございました。 

コメントを確認中で、改めてご連絡させていただきます。 

敬具 
Microsoft DOCS International Team

**Comment by kiyokura** [comment]
I think these prefixes (DO/CONSIDER/AVOID/DO NOT) should not be translated in this case.
When translating it becomes more confusing, because there is no Japanese word suitable as a prefix.
For books published as Japanese translation of "Framework Design Guidelines: Conventions, Idioms, and Patterns for Reusable .NET Libraries, 2nd Edition", these prefixes were left English as it is.

**Comment by LisandroSu** [comment]
こんにちは、@megascus
 
言語に関するフィードバックをお寄せいただきありがとうございました。
 
お寄せいただいたフィードバックの内容を Microsoft   の言語チームで検証いたしましたところ、残念ながら、マイクロソフトのスタイルガイドラインに従わない   という理由で、承認基準に完全には適合していないという結果になりました。
 
記事の品質向上にご協力いただき誠にありがとうございました。引き続きお客様からの貴重なフィードバックをお待ちしています。

敬具
 
Microsoft DOCS International   Team

**Comment by FantomJAC** [comment]
This is totally ridiculous since its translations are absolutely wrong. Also +1 to @kiyokura opinion that it is not suitable to add such prefix within Japanese context.

**Comment by inductor** [comment]
Style guidelines do not define the quality documentation. Please do not take the guideline as "the purpose". The purpose of translation is to make good documentation but not to follow the guideline precisely...

**Comment by ufcpp** [comment]
The problem here is your mistranlation, not the translation guideline issue.

ja-jp
![image](https://user-images.githubusercontent.com/3760955/43111771-9d3855ce-8f2d-11e8-886b-6f9e56d47938.png)

en-us
![image](https://user-images.githubusercontent.com/3760955/43111779-ae935990-8f2d-11e8-8e34-075c063224cf.png)

"しないで" means "DO NOT". You mistranslate "DO" to "DO NOT".

**Comment by heartguard** [comment]
It is strange Japanese translation (and it should be corrected in the future), but I must say "する/DO" and "しないで/DO NOT" does not have same meaning.
In Japanese, "しないで" means it SHOULD NOT to do(it means "X"). "する" means it SHOULD to do(it means "✓"). The meaning between "しないで" and "する" are contrary.
This is not a problem about document quality. Just a translation error.

**Comment by yumetodo** [comment]
Microsoft DOCS International Team decided that `DO` and `DO NOT` has equal mean...
It's really outrageous!

**Comment by AtsushiYamashita** [comment]
Please add the original pages link in these translated pages.
That way, I think this problems will not make us unhappy.
That translation is so difficult for native Japanese people more then original english document.
Best regards!

**Comment by heartguard** [comment]
Here is the English version (original) document.
https://github.com/dotnet/docs/blob/live/docs/standard/design-guidelines/names-of-type-members.md

**Comment by ufcpp** [comment]
@AtsushiYamashita 
![image](https://user-images.githubusercontent.com/3760955/43120095-f78cc03e-8f53-11e8-964a-c5760705a8c7.png)

**Comment by EzoeRyou** [comment]
DO fix the document which has a translation error.
しないで誤訳を含むドキュメントの修正

Do translate with MS style guideline conformance.
しないでMSスタイルガイドライン準拠の翻訳

**Comment by na4zagin3** [comment]
If you want to locate “DO” and “DON'T” in the beginning of sentences in Japanese too, you can translate like “する：” and “しない：” or “正：” and “誤：”. Anyway, it's not possible to literarily translate those words into verb-final languages.

**Comment by nextat** [comment]
why not use Google Translate?
![2018-07-24 16 37 15](https://user-images.githubusercontent.com/29115550/43123897-89cd1f32-8f60-11e8-91ab-5c4b95d7a51b.png)

**Comment by n1215** [comment]
why not use Google Translate?
![google_translate](https://user-images.githubusercontent.com/2157593/43123972-b910a9bc-8f60-11e8-862f-3584112ac458.png)

**Comment by yumetodo** [comment]
@n1215 due to the license

**Comment by YusukeSuzuki** [comment]
翻訳者がGoogle翻訳を使うならライセンスの問題が発生するが読者が自分のブラウザで使う分にはBing翻訳よりもMicrosoft DOCS International Teamに頼るよりもずっと良い選択肢だろう。

**Comment by yoshioterada** [comment]
Dear Cecilia-san.

I'm employee of Microsoft Japan. 
All the above Japanese saying is correct, we should modify the mistranslation.
This is not the "style guide line issue".

Not only mistranslation, the contents was very difficult to understand for native Japanese.
At that point, we should admire all of these Japanese guys. They are excellent reviewer for us.
And we have to hear customers voice.

**Comment by chomado** [comment]
Hi @CeciAc and @LisandroSu ,
Thank you for your great effort on managing Japanese translation :D
I'm also a Microsoft employee, and I recommend reconsidering this issue; you closed this issue because you thought it didn't follow "style- guideline", however, many people here don't think it was a mere guidline matter (including me).
The details are above; Yoshio Terada and other dedicated community members mentioned about it. We appreciate it if you would check them.
If there's anything I can do for you (as an employee same as you), please let me know :D 
Regards,
Madoka
(親切な人たちがたくさん言及してくれてるから、ぜひ今一度 再考をお願いします、と、メンション付きで、メールっぽく書いてみました😊皆さまドキュメントにご指摘ありがとうございます！)

**Comment by runceel** [comment]
Hi @CeciAc, @LisandroSu,
 

I'm Microsoft Japan employee.
As many people said, I think that is miss translation."DO" is "します","DO NOT" is "しないで".

Could you re-open this issue?

**Comment by sakurai-youhei** [comment]
Hi there, I’m never Microsoft employee but I’m helping you to understand why Japanese is so interested in this linguistic quality issue on technical documentation.

The reason is very simple (at least for me); In a few words, the error looks too ugly. Furthermore, (I guess) it’d be even garbage for newbies who are trying to learn something new from literature on the page because it often describes ideas oppositely.

And one more point you’d need to be aware of is that there is almost no way for Japanese to guess what’s written behind by English due to language distance and language barrier. Only one available option is usually moving to other 3rd party technical sources written in Japanese from Microsoft’s 1st party documentation. (Don’t you think this disadvantage is so unfortunate for technical people?)

To sum it up, everyone including me believes Microsoft should be one of the best companies so that we naturally think Microsoft is supposed to keep improving the translation quality in any means.

**Comment by megascus** [comment]
オフトピだし、あえて日本語で書きますが、日本語の説明文書のページに、変なところがあればここにって日本語で案内されたフォームから書いてみると、githubへ勝手に転記され(ここまでは良いが)そこから先はさも当然のようにMS社員とMSMVPというマイクロソフトに近い人たちがさも当然のように英語で議論を始めるってのは外部者としては全く理解のできない文化です。。。

**Comment by gosali** [comment]
Hello @megascus @yoshioterada @chomado @runceel and everyone involved in this issue,

Thank you all for your feedback and sorry for all the back and forth.

Our linguistic team is reviewing your linguistic suggestions in this Pull Request: https://github.com/dotnet/docs.ja-jp/pull/134/files
We will implemented your feedback and published to the updated articles on the Docs site to the following page: https://docs.microsoft.com/ja-jp/dotnet/standard/design-guidelines/names-of-type-members.  

We truly appreciate your contribution to our articles and we encourage you to continue providing your valuable feedback. 

The page in question has been machine-translated, which is not always perfectly understandable. We recently added a "Read in English" option on the right which allows you to switch to the English version without having to change the language settings. But if you feel like the most of the page should be Human translated you can edit the article and generate a pull request with the desire changes and our linguistic team will analyze and implement them.

Kind regards, 
Microsoft DOCS International Team

**Comment by ryokaji** [comment]
@gosali @LisandroSu 
For the future contributions, I hope we could see a guideline when we feedback on the following form.

![ms](https://user-images.githubusercontent.com/9446273/43169183-17dd5e84-8fdb-11e8-90c1-7a129ab5efaf.png)

If you aren't a fluent Japanese speaker or can't read, it would be good to suggest to us that we should report in English again.
As you can see, there are a lot of people understand English, someone would translate it if the author couldn't do.

**Comment by itukikikuti** [comment]
このissueのせいぞろい感が面白い
Google翻訳使えばいいっていうのは問題すり替えすぎじゃないですか？

**Comment by yumetodo** [comment]
@ryokaji It's a good idea. However, feedback form system is not part of this repo, I think...

@itukikikuti クソリプは忘れるべし

**Comment by inductor** [comment]
Google翻訳の件は本人のTwitterに知ってるわそんなことみたいなこと書いてあったので真面目に取り合うだけ無駄なんじゃないかなって思います。

**Comment by ryokaji** [comment]
@yumetodo yeah, that is probably the root cause, but I think that is another topic, so I created #136  last night.
That was immediately closed though :p

**Comment by yumetodo** [comment]
@ryokaji Hmm... Let’s wait a responce from Microsoft DOCS International Team.

**Comment by gosali** [comment]
Hello @megascus @yoshioterada @chomado @runceel and everyone involved in this issue,

Our linguistic team reviewed your linguist suggestions and accepted some of them in this Pull Request: https://github.com/dotnet/docs.ja-jp/pull/134/files. 
We implemented your feedback and We will  publish  the updated articles on the Docs site to the following pages https://docs.microsoft.com/ja-jp/dotnet/standard/design-guidelines/names-of-type-members. 

We truly appreciate your contribution to our articles and we encourage you to continue providing your valuable feedback.  

Kind regards, 
Microsoft DOCS International Team

**Comment by runceel** [comment]
@gosali 
I really appreciate you that you re-opened the issue and reviewing the PR.
Because I think that machine translation is not good yet, I think feedbacks like this is very important.

Thank you.

**Comment by yoshioterada** [comment]
@gosali  Thank You so much for your support for this issue.
I'm very appreciate your help.

Best Regards.
Yoshio.

**Comment by yumetodo** [comment]
#136 was totally ignored even if that was moved to this issue.

I'm going to create new issue to propose that disable feedback form temporary to avoid to make victims (like @megascus ) again. However, I'm fighting with term examinations ( I'm a student of Tokyo University of Science) and have no time to propose. So, When @ryokaji or other people have another plan about the system, DO NOT mind me, DO your plan.

---

I think discussion or notification should not write on twitter. Why don't you write it on this issue? 
This issue status(opened or closed) should not block your action to write here, isn't it?

ref:
- https://twitter.com/yoshioterada/status/1022326473447759872
- https://twitter.com/chomado/status/1022337199407157253
- https://twitter.com/yumetodo/status/1022338124444094464
- https://twitter.com/chomado/status/1022338722530910208

**Comment by ryokaji** [comment]
We shouldn't refer to what we talked on twitter.
![](https://78.media.tumblr.com/b09839524778af3767b4dcff9cd6cdfa/tumblr_p7uf9wn7T11uyg4r8o2_540.gif)

**Comment by gosali** [comment]
Hello@runceel, @yumetodo 

For specific linguistic changes we recommend to submit Pull Request by clicking on the Edit button at the top of the website.
You can there modify the website content and then our linguistic team will review the changes and whether accept or reject them.

Kind regards,
Microsoft DOCS International Team

**Comment by runceel** [comment]
@gosali
Thank you for your recomendation. and thank you for replying on the #136.

**Comment by yumetodo** [comment]
>For specific linguistic changes 

I'm not talking about **specific** linguistic.

- https://github.com/dotnet/docs.ja-jp/issues/136#issuecomment-408038982
- https://github.com/MicrosoftDocs/feedback/issues/554#issue-344868689

is what I wanted to say, and the notice, to propose disabling feedback form temporary, is for temporary measures while discussing how to improve the feedback form.

**Comment by ryokaji** [comment]
@yumetodo [MicrosoftDocs/feedback#554 (comment)](https://github.com/MicrosoftDocs/feedback/issues/554#issue-344868689) was created, and it seems much suitable for discussing about #136 .
Probably your proposal should be proposed there.

**Comment by sasanquaneuf** [comment]
@gosali Hi, I saw Japanese version of the page:
https://docs.microsoft.com/ja-jp/dotnet/standard/design-guidelines/names-of-type-members
and it looks like still wrong. (fig.1)
So I clicked the pencil icon at the top of the page, and I was navigated to github. Then, the content looks correctly. (fig.2)
I think some thing is wrong, isn't it?
I worry that the Edit button affects only github.

fig.1
<img width="1280" alt="2018-07-27 1 04 57" src="https://user-images.githubusercontent.com/16289609/43274650-afa9b28e-913a-11e8-9ed3-33ec5c14f7d3.png">

fig.2
<img width="997" alt="2018-07-27 1 05 46" src="https://user-images.githubusercontent.com/16289609/43274653-b35d4a76-913a-11e8-9a0f-fb77a1b62d0b.png">

**Comment by megascus** [comment]
LGTM!

**Comment by sasanquaneuf** [comment]
@nokura @gosali 
I confirmed the newest pages.

- The page of docs is fixed as @nokura said.
- The page of github is not as same as docs.

I think that there are some difference between docs and github.
So, the Edit button of docs may not work good.

regards,

**Comment by gosali** [comment]
Hello @sasanquaneuf ,

That is because the page has not been re-published yet. But the content in GitHub is fixed.
Updates in Live Web Site would be available soon.

Thank you a lot for your feedback.
Microsoft DOCS International Team

**Comment by getnit69** [comment]
> doが"しないで"と翻訳されています。
> 
> ---
> #### ドキュメントの詳細
> 
> ⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*
> 
> * ID: 1cd21cac-a7e8-13ca-d940-3447312871ca
> * Version Independent ID: 7f3307e9-5563-e28f-cdad-9b6e33358c6b
> * Content: [型のメンバーの名前](https://docs.microsoft.com/ja-jp/dotnet/standard/design-guidelines/names-of-type-members)
> * Content Source: [docs/standard/design-guidelines/names-of-type-members.md](https://github.com/dotnet/docs.ja-jp/blob/live/docs/standard/design-guidelines/names-of-type-members.md)
> * GitHub Login: @rpetrusha
> * Microsoft Alias: **ronpet**

A@

**Comment by JasonCard** [comment]
Hi @getnit69 Thank you for your added comment. I found that previous fixes were lost due to source updates and machine translation, so I'll have this topic and similar one human translated.
